lib/stronger_parameters/constraints/file_constraint.rb

```diff
@@ -0,0 +1,13 @@
+require 'stronger_parameters/constraints'
+
+module StrongerParameters
+  class FileConstraint < Constraint
+    def value(v)
+      if [File, StringIO].include? v.class
```

Comment:
should also work with `Rack::Test::UploadedFile` ... so maybe respond_to some methods ...

https://github.com/brynary/rack-test/blob/master/lib/rack/test/uploaded_file.rb